pyproject.toml

```
@@ -58,7 +58,7 @@ airflow-worker = [
     "xstac",
     "pystac",
     "rasterio",
-    "rio-stac",
+    "rio_stac",
```

```suggestion
    "rio-stac",
```
I think this should be a hyphen not an underscore?

---

Sorry, I was wrong about that. Looks like underscore is right.